### ISSUE TITLE
Train my own data

### ISSUE THREAD

**Issue by mauione1** [shicai/MobileNet-Caffe#63]
Hi,
I am new to MobileNet. Could some one help me to train my own data for caffe. I dont see any training script here.

**Comment by qiuxiaqing** [comment]
I also want to know how to generate the new caffemodle with my own database. Have you solved this problem? if yes, would you like to give me some suggestion? Thank you a lot

**Comment by HilmiK** [comment]
You should modify these deploy.prototxt file to create train.prototxt where some examples are given in original caffe repository such as alexnet, vgg16. You can also convert your data into lmdb file.

**Comment by qiuxiaqing** [comment]
yes thank you. you are right! now, I have do this ,but i want to know what's your mean_value in train.prototxt. (shicai's mean_value: [103.94, 116.78, 123.68]......then, must I compute my data's mean_value in this part???)  thank you 
can you tell me how to compute the mean_value? Thank you so much

**Comment by HilmiK** [comment]
You can leave them as they are. They should be computed from Imagenet and should not matter a lot to change them. U can even comment them, the network should still work.

**Comment by qiuxiaqing** [comment]
OK， That may not be what I want . Thank you all the same
